### PR TITLE
add: homepage

### PR DESCRIPTION
**Pull request by nayyyhaa**
Added Homepage screen for E-commerce App.

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks great. Loved the UI

**Review by anik31 — APPROVED** [review]
Amazing work. UI is eye catching & code is also clean.

File: styles/components/footer.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#connect {
+  min-height: 30vh;
+  color: white;
+  background: var(--COLOR-PRIMARY-XXDARK);
+  text-align: center;
+}
```
Comment (anik31) [inline styles/components/footer.css]: just a question, why have you used id here ?

Comment (nayyyhaa) [reply styles/components/footer.css]: I was thinking to scroll to the footer via clicking it on the header(connect button) but, didn't add that functionality though. Will change it back to class

**Comment by nayyyhaa** [comment]
Thanks @ApurvaSawant11 & @anik31 for reviewing. I've made the said changes, now merging the code.

**Pull request merged**